cmake/inputs/patches/ep_spdlog/spdlog.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/include/spdlog/logger-inl.h b/include/spdlog/logger-inl.h
+index d01c08da..5cd01176 100644
+--- a/include/spdlog/logger-inl.h
++++ b/include/spdlog/logger-inl.h
+@@ -250,7 +250,7 @@ SPDLOG_INLINE void logger::err_handler_(const std::string &msg)
+ #if defined(USING_R) && defined(R_R_H) // if in R environment
+         REprintf("[*** LOG ERROR #%04zu ***] [%s] [%s] {%s}\n", err_counter, date_buf, name().c_str(), msg.c_str());
+ #else
+-        std::fprintf(stderr, "[*** LOG ERROR #%04zu ***] [%s] [%s] {%s}\n", err_counter, date_buf, name().c_str(), msg.c_str());
++        std::fprintf(stderr, "[*** LOG ERROR #%04lu ***] [%s] [%s] {%s}\n", (unsigned long)err_counter, date_buf, name().c_str(), msg.c_str());
```

Comment:
Really strange the msys version of fprintf doesn't recognise a standard forward.  Oh well.  One stylistic nag `(uint64_t)` may be better, `static_cast<uint64_t>(err_counter)` may be best.

Reply:
https://stackoverflow.com/questions/41263896/is-the-zu-specifier-required-for-printf
```
#define __USE_MINGW_ANSI_STDIO 1
```
Reason: MinGW directly links to an old, non-updated system library for C. The define links in a newer version of `stdio` support.

Define could be done on the command line and would avoid patching the code, I'm guessing.

Reply:
@eric-hughes-tiledb first thing I tried but that didn't work unfortunately...